docs/faq.md

```diff
@@ -0,0 +1,72 @@
+# Frequently Asked Questions
+
+- [Is Azure SignalR Service ready for production use?](#production-use)
+- [Client connection is closed with error message "No server available". What does that mean?](#no-server-available)
+- [When there are multiple application servers, are messages from clients sent to all of them or just one of them?](#client-server-mapping)
+- [If one of my application servers is down, will SignalR Service know about it? Will clients be notified too?](#server-down)
+- [Why is exception thrown from my custom `IUserIdProvider` when switching from ASP.NET Core SignalR to Azure SignalR Service SDK?](#limited-context)
+- [Can I configure available transports in Azure SignalR Service like ASP.NET Core SignalR?](#configure-transports)
+
+<a name="production-use"></a>
+## Is Azure SignalR Service ready for production use?
+
+Yes if you are using [ASP.NET Core SignalR](https://github.com/aspnet/SignalR) with Azure SignalR Service.
+The generally-available announcement is at [here](https://azure.microsoft.com/en-us/blog/azure-signalr-service-now-generally-available/).
+
+The support of [ASP.NET SignalR](https://github.com/SignalR/SignalR) is still in *public preview*.
+
+<a name="no-server-available"></a>
+## Client connection is closed with error message "No server available". What does that mean?
+
+This error will happen only when clients are sending messages to SignalR Service.
+
+If you don't have any application server and only use SignalR Service's REST APIs, this is **by design**.
+In serverless architecture, client connections are in **LISTEN** mode and should not send any messages to SignalR Service.
+Read more about [REST API](./rest-api.md) in SignalR Service.
+
+If you have application servers, this means there is no application server connected with your SignalR Service instance.
+Possible root causes are:

```

Comment:
Another possibility: the client sends request too early that server has not yet started the connection to service. This can happen when local debugging the web app with Azure Service. Shall we count this possibility in?